e2e-tests/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.7'
+
+services:
+  designer:
+    container_name: e2e_designer
+    image: touk/nussknacker:${NUSSKNACKER_VERSION-staging-latest}
+    environment:
+      CONFIG_FILE: ${NUSSKNACKER_CONFIG_FILE-/opt/nussknacker/conf/dev-application.conf}
```

Comment:
and here do we set this or use fallback ?

Reply:
:+1: